swo_aws_extension/flows/steps/create_new_aws_environment.py

```
@@ -0,0 +1,111 @@
+                order_id=context.order_id,
+                master_payer_id=get_mpa_account_id(context.order),
+            ),
+            title=CRM_NEW_ACCOUNT_TITLE,
+        )
+        try:
+            response = crm_client.create_service_request(context.order_id, service_request)
+        except CRMError as error:
+            logger.info("%s - Failed to create New Account ticket: %s", context.order_id, error)
+            raise UnexpectedStopError(
+                "Error creating New Account ticket", f"Error details: {error}"
+            ) from error
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)